cmake/find/FindCXX17.cmake

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
Please add credits for the developer of this file

Reply:
I've made this file, but the FindFilesystem.cmake file isn't from me/us I really don't know who originally created it. I just modified round about 20%.

Reply:
You have found the file somewhere on Github if I'm not mistaken? Search your browser history and provide the original link of the file and the username of the author. Then write in it that you based your work on this file.
A quick search resulted in this file, which might be the one you based your work on:
https://github.com/vector-of-bool/neo-io/blob/master/cmake/FindFilesystem.cmake

Reply:
The problem is if we do not credit and somebody sees this we might be blocked from releasing the software on the repositories of a lot of OSes.